assignments/prototypes.js

```diff
@@ -32,6 +60,18 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
+function Humanoid (human){
+  GameObject.call(this, human);
+  CharacterStats.call(this, human);
```

Comment:
if you're calling CharacterStats, you don't also need to call GameObject (because CharacterStats is already inheriting that object)